core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -63,6 +63,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->max(32767)
                     ->defaultValue(E_ALL & ~E_NOTICE & ~E_DEPRECATED & ~E_USER_DEPRECATED)
                 ->end()
+                ->booleanNode('legacy_routing')
+                    ->defaultTrue()
+                    ->info('Disabling legacy routing allows to configure the URL prefix and suffix per root page. However it might not be compatible with older extensions.')
```

Comment:
```suggestion
                    ->info('Disabling legacy routing allows to configure the URL prefix and suffix per root page. However, it might not be compatible with third-party extensions.')
```